### PR TITLE
ASM to Raw. Implements #3

### PR DESCRIPTION
**Pull request by wardlem**
Note: This also includes a significant rewrite of the Raw => ASM converter as well.

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -1,59 +1,41 @@
...
-    if (rawScript.length > 0) {
-        return parseRawScriptHex(rawScript, parsedScript)
-    } else {
-        return parsedScript.substring(0, parsedScript.length -1)
-    }
-}
+const parseScript = require('./parseScript')
+const parseAsm = require('./parseAsm')
 
 // TODO: Could also include a dissasembly https://defuse.ca/online-x86-assembler.htm#disassembly2
```
Comment (JBaczuk) [inline index.js]: I think this PR implements this so we can get rid of this comment.

File: token.js [hunk 2]
```diff
@@ -0,0 +1,70 @@
+const rOps = require('bitcoin-ops/map')
```
Comment (JBaczuk) [inline token.js]: This needs to be updated to use the new scraped data.

**Comment by wardlem** [comment]
> Yeah this is much better. Nice. Just noticed it still used the old bitcoin-ops module.
Yes it does.  I will reconcile the two.

**Comment by wardlem** [comment]
@JBaczuk This has been updated

**Pull request closed without merging**